submarine-cloud-v3/controllers/submarine_virtualservice.go

```diff
@@ -39,6 +39,7 @@ func (r *SubmarineReconciler) newSubmarineVirtualService(ctx context.Context, su
 		r.Log.Error(err, "ParseVirtualService")
 	}
 	virtualService.Namespace = submarine.Namespace
+	virtualService.Spec.Hosts = append(virtualService.Spec.Hosts, submarine.Namespace+".submarine")
```

Comment:
Is this `hosts` a necessary option? 
If the user has deployed an ingress controller (e.g. nginx), my understanding is that it should be possible for the user to define the hostname themselves and configure some https certificate at the same time.

Reply:
I think this only acts as a default value. Users can re-define their hostname using `kubectl edit` after the virtualservice is created. Is this ok?

Reply:
@joshvictor1024 
In that case, is it possible for us to add this configuration to the `Custom Resource`? 
Since `VirtualService` by istio is created by submarine operator, I think we should also have the ability to configure it in CR.

Reply:
@cdmikechen This makes sense. I will add field `host` to the submarine CRD. For a working default value (anything but `"*"`) to be assigned when creating the CR in an arbitrary namespace, this field will be optional and default to `<namespace>.submarine`. When a value is given, then that value overwrites the default one.